.changeset/fine-apples-nail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@vercel/frameworks': minor
+---
+
+Add "blade" framework entry
```

Comment:
I used the same text as [this commit](https://github.com/vercel/vercel/commit/f9d8407866ac95db70dfd961cc51a6d2df233efa).